agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/JvmCompiler.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.init;
+
+import com.microsoft.applicationinsights.agent.bootstrap.diagnostics.PidFinder;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+class JvmCompiler {
+
+  // To disable C2 compiler during Application Insights set-up, for Java >= 9, do the following
+  // things.
+  // Execute with -XX:+UnlockDiagnosticVMOptions -XX:CompilerDirectivesFile=path/jvm-disable-c2.json
+  // Content of jvm-disable-c2.json file:
+  // [
+  //  {
+  //    match: [
+  //      "*::*"
+  //    ],
+  //    c2: {
+  //      Exclude: true
+  //    }
+  //  }
+  // ]
+  private static final String
+      APPLICATIONINSIGHTS_EXPERIMENT_CLEAR_COMPILER_DIRECTIVES_AFTER_INITIALIZATION =
+          "applicationinsights.experiment.clear-compiler-directives-after-initialization";
```

Comment:
```suggestion
          "applicationinsights.experiment.clear_compiler_directives_after_initialization";
```

Reply:
should we follow the same naming convention? i have seen lost of underline instead of dash. See AiSemanticAttributes.java

Reply:
this is a system property name, and not a semantic attribute name